src/main/java/uk/nhs/hee/tis/trainee/sync/mapper/AggregateMapper.java

```diff
@@ -149,6 +159,12 @@ default boolean parseBoolean(String s) {
     return Boolean.parseBoolean(s) || Objects.equals("1", s);
   }
 
+  @Named("mapResponsibleOfficer")
+  default HeeUserDto mapResponsibleOfficer(HeeUser heeUser) {
+    HeeUserMapper heeUserMapper = new HeeUserMapperImpl();
```

Comment:
This is a bit clumsy, but what I have found is that you can't annotate this mapper class with 'uses = HeeUserMapper.class' if the HeeUserMapper has a ComponentModel.SPRING annotation, it will be null in unit tests. I tried a bunch of different ways of sorting out the tests and this seemed the least disruptive, but I could be missing something!

Reply:
We've got around those sorts of issues with Reflection before

```java
mapper = new TraineeDetailsMapperImpl();
Field field = ReflectionUtils.findField(TraineeDetailsMapperImpl.class, "traineeDetailsUtil");
assert field != null;
field.setAccessible(true);
ReflectionUtils.setField(field, mapper, new TraineeDetailsUtil());
```